keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlDeliveryConfigRepository.kt

```diff
@@ -36,6 +38,52 @@ class SqlDeliveryConfigRepository(
 
   private val mapper = configuredObjectMapper()
 
+  override fun deleteByApplication(application: String) {
+    jooq.deleteFrom(DELIVERY_CONFIG)
```

Comment:
Shouldn’t all these deletes be executed in a transaction?

Reply:
I used the same pattern as other the delete methods - I'll check if I've missed something.

Reply:
This particular query pattern isn't a good fit for a transaction, we don't want to elongate the period of time within which some of these tables will locked to inserts.